Source/EasyNetQ.Hosepipe.Tests/QueueRetrievalTests.cs

```diff
@@ -33,25 +34,25 @@ public void TryGetMessagesFromQueue()
 
         [Fact]
         [Traits.Explicit("Requires a RabbitMQ server on localhost")]
-        public void PublishSomeMessages()
+        public async Task PublishSomeMessages()
         {
             var bus = RabbitHutch.CreateBus("host=localhost");
```

Comment:
Do you mind adding `using var bus?` as you did in other places?

Reply:
Done